NFIQ2/NFIQ2Algorithm/include/NFIQ2Algorithm.h

```diff
@@ -56,6 +56,21 @@ namespace NFIQ
         bool bOutputSpeed,
         std::list<NFIQ::QualityFeatureSpeed>& qualityFeatureSpeed );
 
+      /**
+       * @brief
+       * Computes the quality feature vector from the input fingerprint image
+       * data.
+       *
+       * @param rawImage
+       * Fingerprint image in raw format
+       *
+       * @return
+       * Computed quality feature fector
+       */
+      std::list<NFIQ::QualityFeatureData> 
+        computeQualityFeatures(
+          const NFIQ::FingerprintImageData& rawImage );
```

Comment:
`astyle` defect tagged by Travis CI:
```suggestion
        const NFIQ::FingerprintImageData& rawImage );
```

Reply:
I wasn't aware that you already activated astyle. I was looking for your commit to check in the initial astyled files.